sandbox.config.json

```diff
@@ -1,3 +1,8 @@
 {
-"template": "node"
+  "template": "node",
+  "infiniteLoopProtection": true,
```

Comment:
other than `"template": "node",`, the rest of this is optional, 
The defaults for these are false, and I figured that specifying these particular protections against a stale preview and infinite loops would be good UX.

Specifying the port is a hint to codesandbox and maybe the dev using the sandbox template -- people new to ember probably wouldn't know what the default port it, as this specification here provides that info